instance/models/database_server.py

```diff
@@ -198,13 +213,112 @@ def default_port(self):
         return MYSQL_SERVER_DEFAULT_PORT
 
 
+class MongoDBReplicaSetManager(models.Manager):
+    """
+    Custom manager for the DatabaseServer model.
+    """
+    def _get_setting(self, field):
+        """
+        Get setting using prefix
+        """
+        return getattr(
+            settings,
+            self.model.DEFAULT_SETTINGS_NAME + '_' + field,
+            None
+        )
+
+    def get_replica_set_settings(self):
+        """
+        Create dictionary with replica settings
+        """
+        return {
+            'user': self._get_setting('USER'),
+            'password': self._get_setting('PASSWORD'),
+            'name': self._get_setting('NAME'),
+            'primary': self._get_setting('PRIMARY'),
+            'hosts': self._get_setting('HOSTS'),
+            'port': self._get_setting('PORT')
+        }
+
+    def _create_default(self):
+        """
+        Create the default database replica set configured in the Django settings, if any.
+        """
+        replica_settings = self.get_replica_set_settings()
+        for setting in replica_settings:
+            if setting != 'port' and replica_settings[setting] is None:
+                logger.error(
+                    "Error creating the default servers for the replica set, please ensure that"
+                    " all needed settings are configured."
+                )
+                return
+
+        replica_set_hosts = replica_settings['hosts'].split(',')
```

Comment:
@rocioar nit: might be good to trim the splitted values - people tend to put spaces after commas :)

I meann, might be good to support both:

* mongo1,mogno2,mongo3
* mongo1, mongo2, mongo3